.github/workflows/release-master.yml

```diff
@@ -5,55 +5,61 @@ on:
   push:
     branches:
       - master
+    tags:
+      - v*
 
 jobs:
   golint:
-    name: vet fmt and lint
-    runs-on: ubuntu-20.04
+    name: Lint
+    runs-on: ubuntu-latest
```

Comment:
I have mixed feelings about this.

In general, it is better to pin to a version, so our CI does not break when GitHub decides to make `ubuntu-latest` point to a newer version.

In this particular case, if CI breaks it is not a big deal: we will realise and come back to fix it. And since this pipeline is not a live system, it is not that time-sensitive.

There is still the risk that a change in `ubuntu-latest` breaks something but does not make our CI fail. Anyway, I think the risk is low and with `latest` we gain auto-updates, so let's give it a try.

Reply:
I don't mind switching it back to a fixed version if we prefer that. I've been using `ubuntu-latest` in most actions and I haven't had anything break yet. We can always refix should something come up